src/resolvers/Chat/admins.ts

```
@@ -1,14 +1,21 @@
-import type { ChatResolvers } from "../../types/generatedGraphQLTypes";
+import { InterfaceUser, User } from "../../models";
+import type { ChatResolvers, ResolverTypeWrapper } from "../../types/generatedGraphQLTypes";
+
 /**
  * This resolver function will fetch and return the list of all chat admins from the database.
  * @param parent - An object that is the return value of the resolver for this field's parent.
  * @returns An `object` that contains the list of users.
  */
-export const admins: ChatResolvers["users"] = async (parent) => {
-  return await User.find({
+export const admins: ChatResolvers["users"] = async (parent, args, context, info): Promise<ResolverTypeWrapper<InterfaceUser>[]> => {
```

ESLint

[error] 'args' is defined but never used.

(@typescript-eslint/no-unused-vars)